packages/sado-connect/src/components/SadoConnectKit.tsx

```diff
@@ -27,11 +27,11 @@ export function SadoConnectKit({
 
   return (
     <>
-      {address === null ? (
+      {address.ordinals === null ? (
```

Comment:
don't you need to keep the check if `address` exists? if `address` is undefined then it will render `PostConnectButton`?

Reply:
yes, for the default is good, but if someone `setAddress` to `null` then this will be wrong. I think regardless of default we should always safeguard against this `null`

Reply:
or we can switch the condition to be safer by switching the `PreConnectButton` and `PostConnectButton` code to make sure if any error/ `false` in the condition will always render `PreConnectButton` instead of `PostConnectButton`